ci/test_standalone_c.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022-2025, NVIDIA CORPORATION.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+
+# Download the standalone C library artifact
+pkg_name="libcuvs_c_${RAPIDS_CUDA_VERSION}.tar.gz.zip"
```

Comment:
```suggestion
pkg_name="libcuvs_c_${RAPIDS_CUDA_VERSION}.tar.gz"
```

I believe the artifact _name_ is suffixed with `tar.gz`, where the actual file contained also has the `.zip` suffix. Fun times.

You can see here the name that has been used with the successful upload: https://github.com/rapidsai/cuvs/actions/runs/19074700862/job/54487081522?pr=1438#step:12:129